pyvrp/_Individual.pyi

```diff
@@ -123,6 +123,16 @@ class Individual:
         int
             Total time warp over all routes.
         """
+    def uncollected(self) -> int:
```

Comment:
Maybe a more descriptive name? This name could as well represent the list of unvisited clients. How about `total_uncollected_prizes`?

Reply:
I've updated this to `prizes()` and `uncollected_prizes()`, which seems fairly clear to me. I'd rather not add `total_` as a prefix, because we also don't do that for the other data attributes on an `Individual`.